journalbeat/install_pkgs.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -x
+
+echo 'Yes, do as I say!' | echo apt-get install -y --force-yes --no-install-recommends $*
```

Comment:
`echo` piped to `echo`? Is this doing something I don't understand? :)

Reply:
indeed, should be `echo '...' | apt-get`